typescript/sdk/src/core/EvmCoreModule.hardhat-test.ts

```diff
@@ -169,6 +170,95 @@ describe('EvmCoreModule', async () => {
   });
 
   describe('Update', async () => {
+    const ismConfigToUpdate: IsmConfig[] = [
+      {
+        type: IsmType.TRUSTED_RELAYER,
+        relayer: randomAddress(),
+      },
+      {
+        type: IsmType.FALLBACK_ROUTING,
+        owner: randomAddress(),
+        domains: {},
+      },
+      {
+        type: IsmType.PAUSABLE,
+        owner: randomAddress(),
+        paused: false,
+      },
+    ];
+    it('should deploy and set a new defaultIsm', async () => {
```

Comment:
@ltyu you've been good about keeping coverage up with new features. Love to see it!

Reply:
+1!